setup.cfg

```diff
@@ -0,0 +1,18 @@
+[metadata]
+name = gym_xiangqi
+version = 0.0.1
+author = Li Yon Tan
```

Comment:
Is it possible to have mutliple authors?

Reply:
LOL I had the same question and I actually did a quick search on that question as I was writing this. But unfortunately I do not think they support multiple authors and I thought it would be most straight forward to put you since our repository is also under your name 😀